Added new solution to Known issues (#110)
Wolox/react-bootstrap: README.md

### Yeoman not found during project generation

> Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. This could be due to having more than one node version installed, or not having the right `PATH`. Either way, here is how to fix this issue.

Reviewer: ```suggestion
Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. This could be due to having more than one node version installed, or not having the right `PATH` configured. Either way, here is how to fix this issue.
```

> NOTE: This solution works only if you have installed node using nvm (node version manager).

Reviewer: I'd make this more noticeable, may be make it __bold__

#### Step 1: Check if you have the right version of node setted

> It's a common issue having two or more versions of node. What you need to do is make sure that the version that is shown with the command `node --version` is the same as the one in your nvm directory (`nvm list`). If not, you can set it using the command `nvm use -version-` (e.g.: `nvm use v13.8.0`).

Reviewer: Jus as a comment, we can define a `.nvmrc` file containing the desired version and `nvm use` without params will take that version. This way we don't need to know which version is the correct one.
https://github.com/nvm-sh/nvm#nvmrc

#### Step 2: Add node binaries folder route to PATH
In the `~/.bashrc`, `~/.zshrc` or the configuration script for your shell you need to add the following line at the end of the file:

```
export PATH=$NVM_DIR/versions/node/$(node --version)/bin:$PATH
```

> Save the changes an then open the console. Run `source .bashrc` to apply the changes of the file. Now run the generator again. It should work just fine.

Reviewer: ```suggestion
Save the changes, close and re-open the console or run `source .bashrc` to apply the changes of the file. Now run the generator again. It should work just fine.
```
